ajax/libs/marker-animate-unobtrusive/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "marker-animate-unobtrusive",
+  "filename": "SlidingMarker.min.js",
+  "title": "SlidingMarker",
+  "version": "0.2.8",
+  "description": "Unobtrusive Google Maps animated marker",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/terikon/marker-animate-unobtrusive.git"
+  },
+  "keywords": [
+    "Google Maps",
+    "marker",
+    "marker animation",
+    "SlidingMarker"
+  ],
+  "author": {
+    "name": "Terikon Apps",
+    "email": "npm@terikon.com",
+    "url": "http://il.linkedin.com/in/romanviskin"
+  },
+  "license": "MIT",
+  "homepage": "https://github.com/terikon/marker-animate-unobtrusive",
+  "npmName": "marker-animate-unobtrusive",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    },
+    {
+      "basePath": "vendor",
+      "files": [
+        "**/*"
+      ]
+    }
```

Comment:
> Is it possible to add vendor folder in addition to dist folder?

In fact you can add both as origin, but I didn't see the files in the commit, so that's the problem.